pkg/util/taints/taints.go

```diff
@@ -311,6 +311,15 @@ func TaintExists(taints []v1.Taint, taintToFind *v1.Taint) bool {
 	return false
 }
 
+func TaintExistsWithEffect(taints []v1.Taint) bool {
```

Comment:
Please make explicit that it checks if the Effect is either NoExecute or NoSchedule, or make it an argument to the function.

Reply:
OK @gmarek , I propose a name like func MatchTaintEffectNoScheduleNoExecute(...) ?